### ISSUE TITLE
Webcam is opening on the flash, as is described in the issue #233

### ISSUE THREAD

**Issue by alissonzampietro** [jhuckaby/webcamjs#239]

**Comment by thiagolenz** [comment]
I have the same problem.

**Comment by jhuckaby** [comment]
What version of Chrome?  What operating system and version?  Do you actually have a webcam attached that you know works in other applications?

- Try this HTML5 Webcam test page: https://simpl.info/getusermedia/
- And this one: https://davidwalsh.name/demo/camera.php
- Try this Flash Webcam test page: https://www.onlinemictest.com/webcam-test-in-adobe-flash
- Try jQuery Webcam: http://www.xarg.org/project/jquery-webcam-plugin/
- Try JpegCamera: https://github.com/amw/jpeg_camera

Do any of those work with your webcam?

**Comment by alissonzampietro** [comment]
Yes, all of them is working perfectly. Well, this problem started happens yesterday, the strange thing is that i didn't update the project or did something similar. Thank you for your help @jhuckaby

**Comment by jhuckaby** [comment]
That is really bizarre.  I have no idea why it would have suddenly stopped working for you.  Does this WebcamJS demo page work for you as well?

https://pixlcore.com/demos/webcamjs/demos/basic.html

Did you recently upgrade your Chrome?  Using HTTPS to access your webapp?  They recently locked all webcam access to HTTPS only.

Good luck, and sorry you are having issues with the library!

**Comment by thiagolenz** [comment]
Suddenly stopped works in Chrome in a mac version, but I also tried in ubuntu with a new instalation of Chrome. I have a deadline to deliver a software. I'm afraid because they'll test in a tablet. It was working untill yesterday, If doesn't have solution I'll change this library to another.

**Comment by alissonzampietro** [comment]
Maybe should be the locked of webcam on https. Well, i'll try to solve this. Thank you very much!

**Comment by rafaelcremer** [comment]
I had the same problem and it was because I had it open first on Internet Explorer... it seems like you can't enable the webcam in two different browsers at the same time. Once I closed IE and refreshed Chrome it worked!
